lms/static/js/student_account/views/RegisterView.js

```diff
@@ -76,18 +83,18 @@
                 saveError: function(error) {
                     $(this.el).show(); // Show in case the form was hidden for auto-submission
                     this.errors = _.flatten(
-                    _.map(
-                        // Something is passing this 'undefined'. Protect against this.
-                        JSON.parse(error.responseText || '[]'),
-                        function(error_list) {
-                            return _.map(
-                                error_list,
-                                function(error) { return '<li>' + error.user_message + '</li>'; }
-                            );
-                        }
-                    )
-                );
-                    this.setErrors();
+                        _.map(
+                            // Something is passing this 'undefined'. Protect against this.
+                            JSON.parse(error.responseText || '[]'),
+                            function(errorList) {
+                                return _.map(
+                                    errorList,
+                                    function(e) { return '<li>' + e.user_message + '</li>'; }
```

Comment:
The lint tests complained about this being called 'error' because error is already defined further up in the scope.